docs/standard/security-analyzer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The .NET Security Analyzers - .NET
+description: Learn how to use the .NET Security Analyzers in the .NET Desktop Analyzers package to find and address security risks
+keywords: .NET, .NET Core
+author: billwagner
+ms.author: billwagner
+ms.date: 01/10/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# The .NET Security Analyzers
+
+You can use the .NET Security Analyzers to find potential security risks
+in your code. These analyzers will find potential issues, and suggest
+fixes to those issues.
+
+The analyzers run interactively in Visual Studio, as you write your code,
+or as part of a CI build. You'll want to add the analyzer to your project as
+early as possible in your development. The sooner you find any potential issues
+in your code, the easier it will be to fix. However, you can add it at any time
+in the development cycle. It will find any issues with the existing code, and
+will start warning about new issues as you keep developing.
+
+## Installing and configuring the .NET Security Analyzer
+
+The .NET Security Analyzers must be installed as a NuGet package on every
+project where you want them to run. Only one developer needs to add them
+to the project. The analyzer package is a project dependency, and will run
+on every developer's machine once they have the updated solution.
+
+The .NET Security Analyzers are part of the [Microsoft.DesktopAnalyzers](https://www.nuget.org/packages/Desktop.Analyzers/)
+NuGet package. This package provides a smaller set of analyzers
+that includes the security analyzers. In most cases, you'll want
+the [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers) NuGet package. 
+The FxCopAnalyzers package contains many more analyzers that
+help your team enforce the rules recommended by FxCop.
+
+To install it, right-click on the project, and select "Manage Dependencies".
+From the NuGet explorer, search for "Desktop Analyzers", or if you prefer,
+"Fx Cop Analyzers". Install the latest stable version in all projects in your
+solution.
+
+Once the NuGet package is installed run a build, and you'll see a list of
+warnings for the three security related issues found by these analyzers:
```

Comment:
what is the analyzer running against to produce these warnings? Does it come with a project that produces the warnings?

Reply:
I re-wrote it to be more clear.